src/libraries/tests.proj

```diff
@@ -151,6 +151,10 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Private.Xml.Linq\tests\xNodeReader\System.Xml.Linq.xNodeReader.Tests.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetOS)' != 'Android'">
+    <ProjectExclusions Include="..\mono\netcore\sample\Android\AndroidSampleApp.csproj" />
```

Comment:
I am curious to know the reason for adding the Android sample here. Because I had an impression that the mono samples are used as a sanity check for mono runtime. Follow that logic, it felt more natural to make the Android sample part of runtime tests. However, I also understand that it is not quite clear/trivial to make Android sample running as part of runtime tests.

Reply:
We went back and forth a bit on this.  We felt that tapping into the libraries testing infrastructure gave us the most maintainable path.